plugins/sinks/columbus/sink.go

```diff
@@ -0,0 +1,92 @@
+package columbus
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/odpf/meteor/core"
+	"github.com/odpf/meteor/core/sink"
+	"github.com/odpf/meteor/utils"
+)
+
+type httpClient interface {
+	Do(*http.Request) (*http.Response, error)
+}
+
+type Config struct {
+	Host string `mapstructure:"host" validate:"required"`
+	Type string `mapstructure:"type" validate:"required"`
+}
+
+type Sink struct {
+	config Config
+	client httpClient
+}
+
+func New(c httpClient) core.Syncer {
+	sink := &Sink{client: c}
+	return sink
+}
+
+func (s *Sink) Sink(ctx context.Context, configMap map[string]interface{}, in <-chan interface{}) (err error) {
+	if err = utils.BuildConfig(configMap, &s.config); err != nil {
+		return sink.InvalidConfigError{}
+	}
+
+	for data := range in {
+		if err = s.sink(data); err != nil {
+			return
+		}
+	}
+
+	return
+}
+
+func (s *Sink) sink(data interface{}) (err error) {
```

Comment:
Let's rename it from sink to something else maybe send it to avoid confusion with Sink method.